src/style.css

```diff
@@ -1,3 +1,72 @@
+:root {
+  --menu-height: 60px;
+  --menu-width: 1024px;
+}
+
 body {
   margin: 0;
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  width: var(--menu-width);
```

Comment:
Do not set header width, this layout is supposed to behave the same on a bigger screens as well